bash_completion.sh

```diff
@@ -239,7 +239,7 @@ _buildtest ()
     config|cg)
 
 
-      local cmds="-h --help co compilers e edit ex executors p path profiles systems validate v view"
```

Comment:
please get rid of `remove` from this line. This is tab completion for `buildtest config` command. This command is under `buildtest config profiles remove`